### PR TITLE
Add VIA support to Tokyo60.

### PR DESCRIPTION
**Pull request by Zambumon**
This PR adds VIA support for the Tokyo60 Rev.1. In addition, the Tokyo60 Readme.md has been updated.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [X] Keyboard (addition or update)
- [X] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [X] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/tokyo60/tokyo60.h [hunk 1]
```diff
@@ -1,7 +1,9 @@
-#ifndef TOKYO60_H
-#define TOKYO60_H
+#pragma once
 
 #include "quantum.h"
+#include "via.h"
```
Comment (noroadsleft) [inline keyboards/tokyo60/tokyo60.h]: This file is automatically included if `VIA_ENABLE = yes`; you shouldn't need it here.

```suggestion
```

File: keyboards/tokyo60/readme.md [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
 ​
-Minimal hacker style mechanical keyboard designed in Tokyo. Full aluminum hi-pro construction with integrated top-plate. Cherry MX compatible. More Info at [Tokyo Keyboard](http://tokyokeyboard.com).  
+Minimal hacker style mechanical keyboard designed in Tokyo. Full aluminum hi-pro construction with integrated top-plate. Cherry MX compatible. More Info at [Tokyo Keyboard](http://tokyokeyboard.com).
 ​
-Keyboard Maintainer: [Tokyo Keyboard](http://tokyokeyboard.com)  
-Hardware Supported: tokyo60 rev1  
-Hardware Availability: [Tokyo Keyboard](http://tokyokeyboard.com), [Massdrop](https://www.massdrop.com/buy/massdrop-x-tokyo-keyboard-tokyo60-keyboard-kit?mode=guest_open)   
+Keyboard Maintainer: [Tokyo Keyboard](http://tokyokeyboard.com)
+Hardware Supported: Tokyo60 Rev.1
+Hardware Availability: [Drop](https://drop.com/buy/massdrop-x-tokyo-keyboard-tokyo60-keyboard-kit?mode=guest_open)
```
Comment (noroadsleft) [inline keyboards/tokyo60/readme.md]:
```suggestion
* Keyboard Maintainer: [Tokyo Keyboard](http://tokyokeyboard.com)
* Hardware Supported: Tokyo60 Rev.1
* Hardware Availability: [Drop](https://drop.com/buy/massdrop-x-tokyo-keyboard-tokyo60-keyboard-kit?mode=guest_open)
```

File: keyboards/tokyo60/keymaps/via/keymap.c [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+        KC_LSFT,            KC_Z,   KC_X,   KC_C,   KC_V,   KC_B,   KC_N,   KC_M, KC_COMM,  KC_DOT, KC_SLSH,      KC_RSFT, MO(FN), \
+             KC_LALT, KC_LGUI,              KC_SPC,                         KC_RGUI, KC_RALT ),
+
+[FN]= LAYOUT_60_hhkb(
+       KC_PWR,   KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,  KC_F10,  KC_F11,  KC_F12,  KC_INS,  KC_DEL, \
+       KC_CAPS, RGB_TOG, RGB_MOD, RGB_RMOD, BL_TOGG, BL_STEP, KC_TRNS, KC_TRNS, KC_PSCR, KC_SLCK, KC_PAUS,   KC_UP, KC_TRNS, KC_TRNS,  \
+       KC_TRNS, KC_VOLD, KC_VOLU, KC_MUTE, KC_EJCT, KC_TRNS, KC_PAST, KC_PSLS, KC_HOME, KC_PGUP, KC_LEFT, KC_RGHT,       KC_PENT,       \
+       KC_TRNS,       KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_PPLS, KC_PMNS,  KC_END, KC_PGDN, KC_DOWN,       KC_TRNS, KC_TRNS, \
+             KC_TRNS, KC_TRNS,                   KC_TRNS,                         KC_TRNS, KC_TRNS ),
+};
```
Comment (noroadsleft) [inline keyboards/tokyo60/keymaps/via/keymap.c]: VIA expects there to be four layers in the keymap by default.

```suggestion

[2]= LAYOUT_60_hhkb(
       _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,  _______,  _______,  _______,  _______,  _______, \
       _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,   _______, _______, _______,  \
       _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,       _______,       \
       _______,       _______, _______, _______, _______, _______, _______, _______,  _______, _______, _______,       _______, _______, \
             _______, _______,                   _______,                         _______, _______ ),

[3]= LAYOUT_60_hhkb(
       _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,  _______,  _______,  _______,  _______,  _______, \
       _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,   _______, _______, _______,  \
       _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,       _______,       \
       _______,       _______, _______, _______, _______, _______, _______, _______,  _______, _______, _______,       _______, _______, \
             _______, _______,                   _______,                         _______, _______ ),
};
```

File: keyboards/tokyo60/config.h [hunk 4]
```diff
@@ -1,15 +1,14 @@
...
-#define VENDOR_ID       0xFEED
-#define PRODUCT_ID      0x6060
+#define VENDOR_ID       0x5436 // "T6"
+#define PRODUCT_ID      0x6060 // Tokyo60
 #define DEVICE_VER      0x0001
 #define MANUFACTURER    Tokyo Keyboard
-#define PRODUCT         tokyo60
-#define DESCRIPTION     q.m.k. keyboard firmware for tokyo60
+#define PRODUCT         Tokyo Keyboard Tokyo60
+#define DESCRIPTION     Tokyo Keyboard Tokyo60
```
Comment (noroadsleft) [inline keyboards/tokyo60/config.h]: Why all the redundancy here?

**Comment by zvecr** [comment]
Thanks!

**Pull request merged**